src/index.html

```diff
@@ -6,13 +6,58 @@
       name="viewport"
       content="width=device-width, initial-scale=1.0"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <title>Product cards</title>
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body
+    class="card"
+    data-qa="card"
+  >
+    <img
+      src="./images/imac.jpeg" class="card__img"
+    />
+    <h3 class="card__title">
+      APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+    </h3>
+    <div class="card__code">Product code: 195434</div>
+    <div class="card__ranking">
+      <div class="card__stars">
+        <ul class="stars stars--4">
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+          <li class="stars__star"></li>
+        </ul>
```

Comment:
The usage of `ul` and `li` for stars suggests a list of items, which is semantically correct. However, ensure that each `li` element has a class indicating whether it's an active star (e.g. `stars__star--active`) or not, to reflect the current rating.